includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -290,6 +304,11 @@ public function is_branded_button() {
 	 * @return  string
 	 */
 	public function get_button_branded_type() {
+		// no longer a valid option
```

Comment:
We already checks for feature flag inside `get_button_settings()`. Given this method `get_button_branded_type` and some other ones here are only used by `get_button_settings()`, I think the individual feature_flag check is redundant. 

But if we want to be careful, maybe we should add this check to `is_branded_button()`, `custom_button_selector()` as well.

Reply:
You're right, we don't need this since I put it in `get_button_settings()` - removed in 15a9ddd